### PR TITLE
Added Background introduction  and modified introduction to analogue sim...

### PR DESCRIPTION
**Pull request by MikeBrinson**
...ulation.

### CODE REVIEW

File: source/index.rst [hunk 1]
```diff
@@ -7,11 +7,17 @@
 Welcome to Qucs Help's documentation!
 =====================================
 
+.. row of flags, alphabetical
```
Comment (guitorri) [inline source/index.rst]: This is now removed, flags was a bad idea. One languge can be used in many places

File: source/index.rst [hunk 2]
```diff
@@ -7,11 +7,17 @@
 Welcome to Qucs Help's documentation!
 =====================================
 
+.. row of flags, alphabetical 
+
+|flab_cs| |flab_de| |flag_en| |flab_es| |flab_fr| |flab_pt| |flab_ru| |flab_uk|
```
Comment (guitorri) [inline source/index.rst]: No more flags

File: source/index.rst [hunk 3]
```diff
@@ -34,11 +40,36 @@ Example schematics
     Available online at http://qucs.sourceforge.net/download.html#example
```
Comment (guitorri) [inline source/index.rst]: Add 
.. only :: html

File: source/index.rst [hunk 4]
```diff
@@ -34,11 +40,36 @@ Example schematics
...
+
+.. |flab_es| image:: _static/flagicons/es.png
+            :target: ../html-es/index.html
+
+.. |flab_fr| image:: _static/flagicons/fr.png
+            :target: ../html-fr/index.html
 
-  Indices and tables
-  ==================
+.. |flab_pt| image:: _static/flagicons/br.png
```
Comment (guitorri) [inline source/index.rst]: pt is now pt_BR

File: source/start.rst [hunk 5]
```diff
@@ -101,7 +36,7 @@ Wonderful, give yourself a clap on the shoulder!
 
 .. |image0_FR| image:: _static/fr/qucsmain.png
 
-.. |image0_PT| image:: _static/pt_BR/qucsmain.png
+.. |image0_PT| image:: _static/pt/qucsmain.png
```
Comment (guitorri) [inline source/start.rst]: pt is now pt_BR

**Comment by guitorri** [comment]
Hi Mike.  Please delete the `DEADJOE` file and update your branch. Just do your changes and push again to the same branch on your fork. The pull request will be updated.

**Comment by guitorri** [comment]
@MikeBrinson  I guess we can close this pull request, right?

**Comment by MikeBrinson** [comment]
Hello Guilherme
Yes - please close pull request.
More changes and additions to follow in thenext stage of document  preparation.
Best wishes.
Mike
 Mike Brinson

mbrin72043@yahoo.co.uk

**Pull request closed without merging**